runtime/src/main/java/org/corfudb/util/serializer/Serializers.java

```diff
@@ -41,14 +41,14 @@ public static final ISerializer getDefaultSerializer() {
         serializersMap.put(QUEUE_SERIALIZER.getType(), QUEUE_SERIALIZER);
     }
 
-    private static final Map<Byte, ISerializer> customSerializers = new HashMap<>();
+    private final Map<Byte, ISerializer> customSerializers = new HashMap<>();
```

Comment:
should we still keep it as a map?  can a runtime use >1 custom serializers?

Reply:
I think it's a map now, right?

Reply:
I meant, can we remove the map?  Why should it be a map?